.gitlab-ci.yml

```diff
@@ -309,3 +309,17 @@ cactus_integration:
     - pip install psutil numpy --upgrade
     - toil clean aws:us-west-2:${BUCKET_NAME}
     - time cactus --setEnv SINGULARITY_DOCKER_HUB_MIRROR --batchSystem kubernetes --binariesMode singularity --clean always aws:us-west-2:${BUCKET_NAME} examples/evolverMammals.txt examples/evolverMammals.hal --root mr --defaultDisk "8G" --logDebug --disableCaching false
+
+awsjobstore_us_east_1:
+  stage: integration
+  script:
+    - pwd
+    - virtualenv -p ${MAIN_PYTHON_PKG} venv && . venv/bin/activate && pip install -U pip wheel && make prepare && make develop extras=[all] packages='htcondor awscli'
+    - export TOIL_TEST_INTEGRATIVE=True
+    - export TOIL_AWS_KEYNAME=id_rsa
+    - export TOIL_AWS_ZONE=us-east-1a
+    # This reads GITLAB_SECRET_FILE_SSH_KEYS
+    - python setup_gitlab_ssh.py
+    - chmod 400 /root/.ssh/id_rsa
+    - make test tests=src/toil/test/jobStores/jobStoreTest.py::AWSJobStoreTest
+    - make test tests=src/toil/test/jobStores/jobStoreTest.py::EncryptedAWSJobStoreTest
```

Comment:
This is going to run all the job store tests a second time in a new region, and add a new Kubernetes job for every CI run. Can't we *just* test bucket creation in that region? Maybe if we factor out the bucket creation function?